root/static/scripts/edit/URLCleanup.js

```diff
@@ -3653,6 +3656,38 @@ for (const relUuid of relationshipTypesByEntityType.release) {
         result: false,
       };
     }
+    if (/^(https?:\/\/)?([^\/]+)\.bandcamp\.com\/?$/.test(url)) {
+      return {
+        error: l(
+          `The artist page of Bandcamp normally has no entries
+           for specific releases,
+           so adding these links to a release is currently blocked.
+           Please add this Bandcamp link to the artist instead,
+           if appropriate.`,
+        ),
+        result: false,
+      };
+    }
+    return relRule(url);
+  };
+}
+
+// Disallow https://*.bandcamp.com/ URLs at release group level
+for (const relUuid of relationshipTypesByEntityType.release_group) {
+  const relRule = validationRules[relUuid];
+  validationRules[relUuid] = function (url) {
+    if (/^(https?:\/\/)?([^\/]+)\.bandcamp\.com\/?$/.test(url)) {
+      return {
+        error: l(
+          `The artist page of Bandcamp normally has no entries
+           for specific release groups,
+           so adding these links to a release group is currently blocked.
+           Please add this Bandcamp link to the artist instead,
+           if appropriate.`,
+        ),
```

Comment:
Similar to the release comment, but mentioning release groups instead I guess?

Reply:
There're no release group pages but review pages, should we mention reviews instead?

Reply:
Oh, actually, this is *already* disallowed for RGs, because it auto-selects to review and this is not a review. We could update the validation there to be more specific and say that profiles should be added to artists/labels and album/track pages to releases or recordings, but I'm also ok with just not touching RGs.